### ISSUE TITLE
Voice Stack Modulator with multi stack to one parameter in 4.3 beta 3

### ISSUE THREAD

**Issue by baconpaul** [free-audio/interop-tracker#11]
If you grab Surge XT nightly

- set the waveform to saw
- set up 3 voice stacks
- modulate osc1 pitch with the voice stack modulator
   - voice 1 -0.2
   - voice 2 0
   - voice 3 +0.2
- you only hear two pitches

If i look inside surge with a print statement to see what I'm getting, I only get the modulation on the first note

It seems very much like you are not sending a distinct modulator when two voice stacks bind to the same param id. Did you perhaps suppress based on `param_id` rather than `note_id` somewhere?

**Comment by baconpaul** [comment]
If I put a print out in surge when I get my CLAP_EVENT_PARAM_MOD (the earliest time) I see

```
process_clap_event:599|  pevt->note_id=320 pevt->key=53 pevt->channel=0 pevt->param_id=2791424113 pevt->amount=-0.22
process_clap_event:599|  pevt->note_id=321 pevt->key=53 pevt->channel=0 pevt->param_id=2791424113 pevt->amount=0
process_clap_event:599|  pevt->note_id=322 pevt->key=53 pevt->channel=0 pevt->param_id=2791424113 pevt->amount=0
process_clap_event:599|  pevt->note_id=320 pevt->key=53 pevt->channel=0 pevt->param_id=2791424113 pevt->amount=-0.22
process_clap_event:599|  pevt->note_id=321 pevt->key=53 pevt->channel=0 pevt->param_id=2791424113 pevt->amount=0
process_clap_event:599|  pevt->note_id=322 pevt->key=53 pevt->channel=0 pevt->param_id=2791424113 pevt->amount=0
process_clap_event:599|  pevt->note_id=320 pevt->key=53 pevt->channel=0 pevt->param_id=2791424113 pevt->amount=-0.22
process_clap_event:599|  pevt->note_id=321 pevt->key=53 pevt->channel=0 pevt->param_id=2791424113 pevt->amount=0
process_clap_event:599|  pevt->note_id=322 pevt->key=53 pevt->channel=0 pevt->param_id=2791424113 pevt->amount=0
```

<img width="420" alt="Screen Shot 2022-05-26 at 3 34 00 PM" src="https://user-images.githubusercontent.com/13387561/170563640-c051a78a-5f9e-433f-84a9-b0aef1c4b92e.png">
<img width="474" alt="Screen Shot 2022-05-26 at 3 34 08 PM" src="https://user-images.githubusercontent.com/13387561/170563663-1eec52cf-f409-45e6-8426-678ec1f1bee2.png">

with these two modulations

That's surge at head. To get the printout, rebuild and around line 600 of SurgeSynthProcessor.cpp before the jp->setPolyphonicMod add

```
    _DBGCOUT << _D(pevt->note_id) << _D(pevt->key) << _D(pevt->channel)
                     << _D(pevt->param_id) << _D(pevt->amount) << std::endl;
```

**Comment by abique** [comment]
Should be fixed in bitwig 4.3 beta 4.

**Comment by baconpaul** [comment]
Confirmed fixed in alex 117053
